components/script/dom/htmliframeelement.rs

```diff
@@ -770,6 +773,8 @@ impl IframeLoadEventSteps {
 impl Runnable for IframeLoadEventSteps {
     fn handler(self: Box<IframeLoadEventSteps>) {
         let this = self.frame_element.root();
-        this.iframe_load_event_steps(self.pipeline_id);
+        if !this.cancel_initial_load_event.get() {
```

Comment:
The `IframeLoadEventSteps` runnable is only used for the initial `about:blank` load. For all other cases,  `iframe_load_event_steps` is called directly in `script_thread.rs`